CHANGELOG.rst

```diff
@@ -1,6 +1,8 @@
 Changelog
 =========
 
+- Drop official Python support for 3.7 and 3.8 as GPy is not officially supported for these versions
+- Add convenience method for obtaining elfi samples as `InferenceData`` to be used with `arviz`
```

Comment:
is it intended that this pull request both 1) updates the python version and 2) includes arviz, or should the arviz parts be removed?

Reply:
Removed, thanks

Reply:
ok! then (since the intention was not to include arviz) i think the arviz parts should be removed. i added comments about them now